pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/mergerollup/MergeRollupTaskGenerator.java

```diff
@@ -277,8 +291,14 @@ public List<PinotTaskConfig> generateTasks(List<TableConfig> tableConfigs) {
         }
         // Create delay metrics even if there's no task scheduled, this helps the case that the controller is restarted
```

Comment:
same for this comment, does it pertain only to old code?

Reply:
This apply to the new metric as well, removed "delay" to make it clearer.